packages/backend/server/src/core/workspaces/controller.ts

```diff
@@ -51,7 +51,7 @@ export class WorkspacesController {
     // metadata should always exists if body is not null
     if (metadata) {
       res.setHeader('content-type', metadata.contentType);
-      res.setHeader('last-modified', metadata.lastModified.toISOString());
+      res.setHeader('last-modified', metadata.lastModified.toUTCString());
```

Comment:
seems last modified has a standard format
https://arc.net/l/quote/wradjpwu
better to follow it?